homeassistant/components/aten_pe/config_flow.py

```diff
@@ -0,0 +1,116 @@
+"""Config flow for aten_pe integration."""
+from __future__ import annotations
+
+import logging
+
+from atenpdu import AtenPE, AtenPEError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PORT, CONF_USERNAME
+
+from .const import (
+    CONF_AUTH_KEY,
+    CONF_COMMUNITY,
+    CONF_PRIV_KEY,
+    DEFAULT_COMMUNITY,
+    DEFAULT_PORT,
+    DEFAULT_USERNAME,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class AtenPEConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for aten_pe."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    def __init__(self) -> None:
+        """Initialize the config flow."""
+        self._errors: dict[str, str] = {}
```

Comment:
This doesn't have to be an instance variable? I don't see it being used across methods in this class.